src/analysis/classic/shaman/elemental/modules/Abilities.ts

```diff
@@ -4,8 +4,6 @@ import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
 
 class Abilities extends CoreAbilities {
   spellbook() {
-    const combatant = this.selectedCombatant;
```

Comment:
Did you mean to change the classic file?

Reply:
I certainly did not! Thank you.